priv/templates/service.ex.eex

```diff
@@ -1,7 +1,12 @@
 defmodule <%= @module %>.Service do
-  <%= unless @module_doc? do %>
+  <%= if @comment != "" do %>
+  @moduledoc """
+  <%= @comment %>
+  """
+  <% else %>
   @moduledoc false
   <% end %>
```

Comment:
I disagree a bit here. I don't think we should `@moduledoc false` if there's no comments for a file and `include_docs` is true. For example, I have an application where we use `include_docs=true` because we want the typespecs and structs to be public in the generated documentation.

Reply:
Pushed this too